src/h2/utilities.py

```diff
@@ -13,51 +13,56 @@
 
 from .exceptions import ProtocolError, FlowControlError
 
+
 UPPER_RE = re.compile(b"[A-Z]")
+SIGIL = ord(b':')
+STATUS_HEADER = b':status'
```

Comment:
Why did you introduce the status header as constant here, but none of the other `:`-prefixed headers that get used frequently?

Reply:
Carry over from the function where this was used. Moved back inline for consistency. Pulling all these common headers to globals _may_ help with performance but thoughts for future.